migrations/versions/940_brief_response_award_details.py

```diff
@@ -0,0 +1,34 @@
+"""award_details JSON blob and awarded_at date stamp on BriefResponse
```

Comment:
Can you let me know when this goes out to the staging environment? I'd like to test the new production to staging data migration stuff when there is an actual difference in alembic versions!